pygmt/figure.py

```diff
@@ -216,7 +216,7 @@ def psconvert(self, icc_gray=False, **kwargs):
             **E** means EPS with PageSize command, **f** means PDF, **F** means
             multi-page PDF, **j** means JPEG, **g** means PNG, **G** means
             transparent PNG (untouched regions are transparent), **m** means
-            PPM, **s** means SVG, and **t** means TIFF [Default is JPEG]. To
+            PPM, and **t** means TIFF [Default is JPEG]. To
```

Comment:
Should we add a link to https://docs.generic-mapping-tools.org/6.4/psconvert.html#notes and/or state in this docstring somewhere that "Fortunately, quality SVG graphics can be obtained by first converting to PDF and then install and use the package pdf2svg."?

Reply:
I don't think we should mention SVG, because gs 9.16 was released several years ago, so SVG is never supported in PyGMT since 2017. People who want SVG format should find tools to do the conversion themselves.

Reply:
Ah right, we're at ghostscript 9.54 or something now. Then just removing the svg mention is fine.